First draft workshop content (#3)
observability-lab-cse/observability-lab-workshop: 10-azure-observabity-101/README.md

 - Differentiating between Metrics-based and Logs-based telemetry in Azure. -->

<!-- Azure has a few observability resources that can be used when creating an observability suite for your solution. For now, let us focus on the following two, as they are most commonly used and cover most use cases:

- [Log Analytics Workspace](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-workspace-overview)
- [Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview)
- Insights Agents: [VM Insights Agents](https://learn.microsoft.com/en-us/azure/azure-monitor/vm/vminsights-overview), [Container Insights Agents](https://learn.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-overview), [Network Insights Agents](https://learn.microsoft.com/en-us/azure/network-watcher/network-insights-overview) -->

Let's look at what normally happens when people need observability.

1. Create Application Insights Resource.
2. Pass the connection string of Application Insights to the application.
3. And bam 💥! Automatically, you get your data in beautiful dashboards on Application Insights.

If this is enough for you, great!

> But often these dashboards don't give you all the information about your application you need. So, in cases of creating new more complex dashboards, alerts, or just out of pure curiosity, you want to know where you can find all the observability data to your availability.

Reviewer: ```suggestion
But often these dashboards don't give you all the information about your application that you need. So, in case more complex dashboards, alerts are needed, or just out of pure curiosity, you want to know where you can find all the observability data.
```
Not sure exactly what you mean with 'to your availability' here.
